lib/reporters/tap.js

```diff
@@ -128,6 +138,9 @@ class TapReporter {
 			case 'hook-failed':
 				this.writeTest(evt, {passed: false, todo: false, skip: false});
 				break;
+			case 'hook-finished':
+				this.writeLogs(evt);
```

Comment:
Looking at the verbose reporter, if there are log lines it prints the hook title as well as the logs. Could you do that same check here and use `this.writeTest()` instead?

Reply:
not really cause the logs are counted as tests and they are not. I just want to print the `log`
https://github.com/avajs/ava/pull/2134/files/eb226a2ed7abc81707b059d53a61b59b79d2f343#diff-35f98ab820e4d7c63d4cc39d54e30b5bL102

also this is another problem when my hook fails it does that and the counts turn into a mess:
```
not ok 23 - api › users › createUser › [monitoring, validation] Verify a user is created
not ok 25 - api › users › createUser › Delete test user for [monitoring, validation] Verify a user is created
```
`Delete test user for` is a hook and the counter is increased because of this:
https://github.com/avajs/ava/pull/2134/files/eb226a2ed7abc81707b059d53a61b59b79d2f343#diff-35f98ab820e4d7c63d4cc39d54e30b5bR139

Reply:
The tests count is 24 not 25

Reply:
CC @novemberborn

Reply:
The counting issue aside, the idea of using `writeTest` is that it gives you information about where the log lines came from. So I'd still like to do that.

Reply:
@novemberborn it's done